cypress/integration/example-auto-scroll-when-dragging.spec.js

```diff
@@ -0,0 +1,390 @@
+/// <reference types="cypress" />
+
+describe('Example - Auto scroll when dragging', { retries: 1 }, () => {
+  // NOTE:  everywhere there's a * 2 is because we have a top+bottom (frozen rows) containers even after Unfreeze Columns/Rows
+  const cellWidth = 80;
+  const cellHeight = 25;
+  const gridWidth = 600;
+  const gridHeight = 350;
+  const scrollbarDimension = 17;
+
+  const fullTitles = ['#', 'Title', 'Duration', '% Complete', 'Start', 'Finish', 'Cost', 'Effort Driven'];
+
+  for (var i = 0; i < 30; i++) {
+    fullTitles.push("Mock" + i);
+  }
+
+  it('should load Example', () => {
+    cy.visit(`${Cypress.config('baseExampleUrl')}/example-auto-scroll-when-dragging.html`);
+  });
+
+  it('should have exact column titles on grid', () => {
+    cy.get('#myGrid')
+      .find('.slick-header-columns')
+      .children()
+      .each(($child, index) => expect($child.text()).to.eq(fullTitles[index]));
+    cy.get('#myGrid2')
+      .find('.slick-header-columns')
+      .children()
+      .each(($child, index) => expect($child.text()).to.eq(fullTitles[index]));
+  });
+
+  function dragStart(parentClass, row, col) {
+    cy.get(`${parentClass || ''} [style="top:${row * 25}px"] > .slick-cell:nth(${col})`, )
+      .as('dragStart')
+      .click({ force: true })
+      .trigger('mousedown', { which: 1 })
+      .trigger('mousemove', cellWidth/3, cellHeight/3);
+  }
+
+  function drag(addRow, addCell) {
+    cy.get('@dragStart').trigger('mousemove', cellWidth * (addCell + 0.5) , cellHeight * (addRow + 0.5), { force: true })
+  }
+
+  function dragOutside(selector, position, ms, px) {
```

Comment:
I see a lot of functions in your test file, it might make sense to create a sepate `.js` file to move all these functions outside of the tests (if doable).

Reply:
@aasdkl if you can just fix this last comment, the rest looks ok to me.

Reply:
sorry for late, I've been a little busy these day.
Let me try to extract some common functions into the separate file, maybe can finish in this week.